src/Sarif.Viewer.VisualStudio/Options/SarifViewerOptionPage.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.Sarif.Viewer.Options
+{
+    [ComVisible(true)]
+    public class SarifViewerOptionPage : DialogPage
+    {
+        private const string CategoryName = "Sarif Viewer Options";
+
+        private const string MonitorSarifFolderDisplayName = "Enable loading sarif results from .sarif folder automatically";
```

Comment:
>sarif results [](start = 77, length = 13)

.sarif files

Reply:
fixed

---
In reply to: [602306723](https://github.com/microsoft/sarif-visualstudio-extension/pull/382#discussion_r602306723) [](ancestors = 602306723)